wg-etcd-operator/annual-report-2024.md

```diff
@@ -2,26 +2,25 @@
 
 ## Current initiatives and Project Health
 
-
 1. What work did the WG do this year that should be highlighted?
 
-<!--
-   Some example items that might be worth highlighting:
-   - artifacts
-   - reports
-   - white papers
-   - work not tracked in KEPs
--->
+* We launched the WG, and [recruited contributors](https://github.com/etcd-io/etcd-operator/graphs/contributors) from multiple prior operators
+* Completed [review of leading existing operators](https://github.com/etcd-io/etcd-operator/blob/main/docs/wg/evaluation/evaluation.pdf)
+* [Surveyed potential users](https://github.com/etcd-io/etcd-operator/blob/main/docs/wg/survey/2024-operator-survey.md) and determined [operator development roadmap](https://github.com/etcd-io/etcd-operator/blob/main/docs/roadmap.md)
+* Created initial [operator workflow]()
```

Comment:
The reconciliation workflow is just part of the design, let me deliver a PR to add the design into the repo next week.

```suggestion
* Finished the high-level [design for v0.1.0](to be added)
```

Reply:
or we can link to the google doc directly?